pipeflow_lib.tlv

```diff
@@ -656,72 +627,32 @@ m4_trans_ind            $ANY = |m4_out_pipe/fifo_head['']m4_trans_hier$ANY;
          ?$trans_valid
             $ANY = /read2$ANY;
    */
-'],
-
-['
-m4_popdef(['m4_top'])
-m4_popdef(['m4_in_pipe'])
-m4_popdef(['m4_in_at'])
-m4_popdef(['m4_out_pipe'])
-m4_popdef(['m4_out_at'])
-m4_popdef(['m4_depth'])
-m4_popdef(['m4_trans_hier'])
-m4_pushdef(['m4_high_water'])
-
-m4_popdef(['m4_ptr_width'])
-m4_popdef(['m4_counter_width'])
-m4_popdef(['m4_bypass_align'])
-m4_popdef(['m4_reverse_bypass_align'])
-m4_popdef(['m4_trans_ind'])
-'])
 
 
 // A FIFO using simple_bypass_fifo.
 // Requires include "simple_bypass_fifo.sv".
 //
 // The interface is identical to m4_flop_fifo, above, except that data width must be provided explicitly.
 //
-m4_define_plus(['m4_old_simple_bypass_fifo_v2'], ['
-m4_pushdef(['m4_top'],       ['$5'])
-m4_pushdef(['m4_in_pipe'],   ['$6'])
-m4_pushdef(['m4_in_at'],     ['$7'])
-m4_pushdef(['m4_out_pipe'],  ['$8'])
-m4_pushdef(['m4_out_at'],    ['$9'])
-m4_pushdef(['m4_depth'],     ['$10'])
-m4_pushdef(['m4_width'],     ['$11'])
-m4_pushdef(['m4_trans_hier'],['$12'])  // Eg: ['/flit'] or ['']
-m4_pushdef(['m4_high_water'],['$13'])  // Default to 0.  Number of additional entries beyond full.
-
-'], m4___file__, m4___line__, ['
+\TLV m4_old_simple_bypass_fifo_v2(/m4_top,|m4_in_pipe,@m4_in_at,|m4_out_pipe,@m4_out_at,#m4_depth,#m4_width,#m4_trans_hier,#m4_high_water)
```

Comment:
m4_trans_hier should become /m4_trans_hier

Reply:
sry cannot figure why /m4_trans_hier not #m4_trans_hier